Resources/Prototypes/Entities/Clothing/OuterClothing/wintercoats.yml

```diff
@@ -853,6 +853,11 @@
   - type: Construction
     graph: WebObjects
     node: coat
+  - type: Butcherable
+    butcheringType: Knife
```

Comment:
```suggestion
```
that's already the default value, so we can remove it

Reply:
Removed